components/base/wheeled/wheeled_base.go

```diff
@@ -95,13 +98,28 @@ type wheeledBase struct {
 	right     []motor.Motor
 	allMotors []motor.Motor
 
-	opMgr operation.SingleOperationManager
+	movementSensors   []movementsensor.MovementSensor
+	orientationSensor movementsensor.MovementSensor
+
+	opMgr  operation.SingleOperationManager
+	logger golog.Logger
 }
 
 func (base *wheeledBase) Spin(ctx context.Context, angleDeg, degsPerSec float64, extra map[string]interface{}) error {
 	ctx, done := base.opMgr.New(ctx)
 	defer done()
 
+	if base.orientationSensor != nil {
+		base.logger.Debugf("received a spin with movement sensor: angle: %.2f, speed: %.2f", angleDeg, degsPerSec)
```

Comment:
I really appreciate this debug logging ([opt] maybe add a newline to the end of the strings though?)

Reply:
Need clarification, please.

Reply:
just a "\n" at the end of the string so it doesn't bleed into the next log

Reply:
This logger already does that.